dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/DefaultTrackerImportService.java

```diff
@@ -82,7 +82,7 @@ private PersistenceReport commit(TrackerImportParams params, TrackerBundle track
   @IndirectTransactional
   public ImportReport importTracker(
       TrackerImportParams params, TrackerObjects trackerObjects, JobProgress jobProgress) {
-    User user = trackerUserService.getUser(params.getUserId());
+    User user = trackerUserService.getCurrentUser();
```

Comment:
In the next iteration of this epic, we will replace instances of the `User` class such as this one, with `UserDetails`, correct?

Reply:
Yes, that is the idea.